### ISSUE TITLE
Missing config

### ISSUE THREAD

**Issue by bauerem** [plandes/dsprov#1]
If I follow the instructions in the README.md, I always run into the error of a missing config file.
Would be great if you could provide an example config file or link to an explanation of how to construct this file.

(Also, when installing dependencies with the pip command you provide, an error occurs when installing psycopg2 is missing.
I resolved this by installing libpq-dev but I think this should be explained too.)

**Comment by plandes** [comment]
@bauerem Sorry you're having issues.

Thank you for letting me know about these issues and finding the missing link back to the upstream [zensols.mimic](https://github.com/plandes/mimic) package that has the configuration documentation.  I have added those links and another example in [new section](https://github.com/plandes/dsprov?tab=readme-ov-file#installation-and-configuration) in the documentation.

Re the `psycopg2` module install error: this was resolved by defaulting the SQLite MIMIC-III databases upstream, but I have now released a new package that use this fix so this will not happen to others in the future.  Thanks once again for noting that and let me know if you have any further issues or questions.

**Comment by plandes** [comment]
@bauerem Sorry you're having issues.

Thank you for letting me know about these issues and reminding me about the missing link I omitted that points back to the upstream [zensols.mimic](https://github.com/plandes/mimic) package that has the configuration documentation.  I have added those links and another example in [new section](https://github.com/plandes/dsprov?tab=readme-ov-file#installation-and-configuration) in the documentation.

Re the `psycopg2` module install error: this was resolved by defaulting the SQLite MIMIC-III databases upstream, but I have now released a new package that uses this fix so this will not happen to others in the future.  Thanks once again for noting that and let me know if you have any further issues or questions.
